data/sql_setup/prepare_schedule_a.sql

```diff
@@ -26,24 +26,31 @@ begin
         two_year_transaction_period_new = get_transaction_year(new.contb_receipt_dt, new.rpt_yr);
 
         if two_year_transaction_period_new >= start_year then
+            delete from ofec_sched_a_queue_new where sub_id = new.sub_id;
             insert into ofec_sched_a_queue_new values (new.*, timestamp, two_year_transaction_period_new);
         end if;
+
         return new;
     elsif tg_op = 'UPDATE' then
         two_year_transaction_period_new = get_transaction_year(new.contb_receipt_dt, new.rpt_yr);
         two_year_transaction_period_old = get_transaction_year(old.contb_receipt_dt, old.rpt_yr);
 
         if two_year_transaction_period_new >= start_year then
+            delete from ofec_sched_a_queue_new where sub_id = new.sub_id;
```

Comment:
After a short chat we all determined that the fix as it stands is the appropriate way to go.  In the instance of the one edge case of processing a record with potentially older data than what was previously added to the queues for the nightly update, we decided to go ahead with the processing so that at least the data is consistent with what is in the source tables.

Should this ever be the case, the fix would be for the FEC folks to update the data in the source tables, which will trigger another update on our side to pull in the correct information.  By maintaining consistency, it'll be easier to spot such an issue if it were to ever happen (hopefully not!).

This is now ready for merge!